vignettes/safestats-vignette.Rmd

```diff
@@ -153,142 +157,253 @@ mean(sValues > 1/alpha) >= power
 Due to sampling error, the average number of times that sValue > 1/alpha might not always be larger than the specified power, but it is always close to it. The sampling error decreases as the number of replications increase. 
 
 ### Data under the alternative: <a name = "optStopT">Stopping the experiment early</a>
-What makes safe tests particularly interesting is that they allow for early stopping without the test exceeding the tolerable type I error $\alpha$. This means that the evidence can be monitored as the data comes in, and when there is a sufficient amount of evidence against the null, thus, the s-value larger than $1/\alpha$ that the experiment can be stopped early, which increases efficiency. We highlight this with our design object 
+What makes safe tests particularly interesting is that they allow for early stopping without the test exceeding the tolerable type I error rate of $\alpha$. This means that the evidence can be monitored as the data comes in, and when there is a sufficient amount of evidence against the null, thus, the s-value larger than $1/\alpha$ that the experiment can be stopped early, which increases efficiency. We highlight this with our design object 
```

Comment:
Ik zie niet precies hoe dit gehighlighted wordt, ik zie dit:
 Safe Paired Sample T-Test
 #>
#> Requires an experiment with sample sizes:
 #> n1Plan = 63 and n2Plan = 63
 #> to find an effect size of at least:
 #> deltaMin = 0.42426
 #>
#> with:
 #> power = 0.8 (thus, beta = 0.2)
 #> under the alternative:
 #> true difference in means ('x' minus 'y') is greater than 0
 #>
#> Based on the decision rule S > 1/alpha:
 #> S > 20
 #> which occurs with chance less than:
 #> alpha = 0.05
 #> under iid normally distributed data and the null hypothesis:
 #> mu = 0